tests/utils/disable-download-spec.ts

```diff
@@ -0,0 +1,24 @@
+import { disableDownload } from '../../src/utils/disable-download';
+import { isJest } from '../../src/utils/is-jest';
+
+jest.mock('../../src/utils/is-jest');
+
+describe('disableDownload', () => {
+  // versions below 11.0.0 are not available for download on darwin arm64
+  it('returns false for downloadable versions and true otherwise', () => {
+    if (process.platform == 'darwin' && process.arch == 'arm64') {
```

Comment:
Can we mock `platform` and `arch` here? Otherwise this test only tests the code path if it's run on Apple silicon, which the CI tests won't cover. Let's try to cover both code paths with mocks - don't rely on the specifics of what machine is running the test.